Library/Homebrew/brew.sh

```diff
@@ -136,6 +136,9 @@ then
 elif [[ -n "$HOMEBREW_DEVELOPER" && -x "$HOMEBREW_CURL_PATH" ]]
 then
   HOMEBREW_CURL="$HOMEBREW_CURL_PATH"
+elif [[ -x "$HOMEBREW_CURL" ]]
+then
+  HOMEBREW_CURL="$HOMEBREW_CURL"
```

Comment:
Were these variables not working at all without these changes? If so, I'm wondering if we remove the docs entirely and just keep the `FORCE` variables around. Thoughts @EricFromCanada?

Reply:
Correct; originally, `brew.sh` would overwrite whatever values `HOMEBREW_CURL` and `HOMEBREW_GIT` had in the original environment with its own. I've no opinion either way.

Reply:
@EricFromCanada Yeh, let's just remove these variables I reckon as no-one complained 👍 Thanks and sorry for the back and forth!